docs/faq.md

```diff
@@ -275,7 +275,7 @@ See [UUIDs in JSON files](uuid.md) for more details.
 
 Dollar signs by default denote either **inline** (`$ x $`) or **display mode** (`$$ x $$`) environments.
 
-To escape either math environment, consider using PrairieLearn's markdown tag and inline code syntax.
+To escape either math environment, consider using PrairieLearn's Markdown tag and inline code syntax.
```

Comment:
```suggestion
To escape either math environment, consider using PrairieLearn's `markdown` tag and inline code syntax.
```

Reply:
Changed in [f2429b4](https://github.com/PrairieLearn/PrairieLearn/pull/11643/commits/f2429b43439d5e25b6eb82f254e7d55f94d69384).